docs/theme-context.rst

```diff
@@ -0,0 +1,117 @@
+Read the Docs data passed to Sphinx build context
+=================================================
+
+Before calling `sphinx-build` to render your docs, Read the Docs injects some
+extra context in the templates by using the ``html_context`` Sphinx setting in
+the ``conf.py`` file. This extra context allows Read the Docs to add some
+features like "Edit on GitHub" and use a custom Analytics code, among others.
+
+
+Context injected by default
+---------------------------
+
+Here is the full list of values injected by Read the Docs as a Python
+dictionary. Note that this dictionary is injected under the main key
+`readthedocs`:
+
+
+.. code:: python
+
+    'readthedocs': {
+        'version': {
+            'name': str,
+            'slug': str,
+            'latest': bool,
+            'stable': bool,
+            'downloads': {
+                'pdf: str,
+                'htmlzip': str,
+                'epub': str
+            }
+        },
+        'project': {
+            'name': str,
+            'slug': str,
+            'rtd_language': str,  # can be different than language in conf.py
+            'canonical_url': str,
+            'single_version': bool,
+            'versions': [
+                {
+                    'slug': str,
+                    'href': str
+                }
+            ],
+            'subprojects': [
+                {
+                    'slug': str,
```

Comment:
Similar to above regarding the list of versions, having more subproject fields here would be helpful for making a landing page, for example. This should either be a serialized Project, or at very least should include a pk.